test/letters/containers/AddressSection.unit.spec.jsx

```diff
@@ -46,6 +46,9 @@ const defaultProps = {
   statesAvailable: true
 };
 
+// For running ./node_modules/.bin/mocha directly on this file
```

Comment:
what is doing this for?

Reply:
I prefer running tests using the command `./node_modules/.bin/mocha test/letters/blah.unit.spec.js` instead of adding (and potentially forgetting to delete) an `.only` to the `describe` and running `npm run test:unit`.

The trouble I had, though, is about half the tests failed because `global.window.dataLayer` was `undefined`. Adding this line in here doesn't actually change anything when running `npm run test:unit`, but it did let me run `./node_modules/.bin/mocha ...`.